packages/prosess-beregningsgrunnlag/src/components/fellesPaneler/SkjeringspunktOgStatusPanel.jsx

```diff
@@ -61,6 +62,14 @@ export const SkjeringspunktOgStatusPanelImpl = ({ skjeringstidspunktDato, aktivi
             </Normaltekst>
           </FlexColumn>
         </FlexRow>
+        {lonnsendringSisteTreMan && <>
+          <VerticalSpacer sixteenPx />
+          <FlexRow>
+            <FlexColumn>
+              <BlaBoksMedCheckmark textId="Beregningsgrunnlag.Skjeringstidspunkt.LonnsendringSisteTreMan" />
```

Comment:
Korleis blir dette dersom man har både kortvarig arbeidsforhold og lønnsendring samtidig?

Reply:
Har lagt in stötte for flere texter i checkboksen men avventer med kortvarig arbeidsforhold ref konversation på slack